lists/lists.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Lists</title>
+    <link rel="stylesheet" href="./lists.css">
+</head>
+
+<body>
+    <h1> Lists </h1>
+
+    <h2> Simple List </h2>
+    <section class="container-for-lists">
+        <ul class="list-general list-simple ">
+            <li> India </li>
+            <li> Canada </li>
+            <li> Nepal</li>
+            <li> Sri Lanka </li>
+            <li> Maldieves </li>
```

Comment:
Hey! for these list items, you can add a slight border like this.  The rest of the code is good 👍 
![image](https://user-images.githubusercontent.com/77927918/151914655-5ad4105f-24b4-4931-9804-6986440f97e7.png)

Reply:
I did it and it looks great. Thank you for the suggestion :) 💯